components/page/DevPageYAML.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+    Card,
+    CardBody,
+    CodeBlock,
+    CodeBlockCode,
+    CardHeader
+  } from '@patternfly/react-core'
+
+/**
+ * props for DevFile YAML component
+ * 
+ * @param devYAML - yaml text with /n for each new line
+ */
+interface Props {
+    devYAML:string
+}
+/**
+ * component  for displaying DevFile YAML
+ * 
+ * @remarks
+ *      component only for stacks, won't be present for samples
+ * 
+ * @param props - yaml text
+ */
+const DevPageYAML = ({devYAML}:Props) => {
+    return (
+        <Card 
+            isRounded
+            style ={{width:"75%", maxWidth:"1000px", margin:"auto", marginTop:"30px", marginBottom:"30px"}}>
+            <CardHeader style={{padding:"10px", textAlign:"center"}}>
+                devfile.yaml
+            </CardHeader>            
+            <CardBody style={{padding:"0"}}>
+            <CodeBlock style={{backgroundColor:"#1F1F20", color:"#FFFFFF"}}> 
+                    {/*codeblock rendered with mapping st each line in the yaml file has a corresponding index that is unselectable by user*/}
```

Comment:
typo - codeblock rendered with mapping **_at_** each line

Reply:
oh i meant st as an abbreviation for "such that" but ill change it to the non-abbreviation